### ISSUE TITLE
CHARMM validation cleaning up

### ISSUE THREAD

**Issue by patrickfuchs** [patrickfuchs/buildH#64]
Move CHARMM_POPC.json to there in dir `lipids`.

**Comment by patrickfuchs** [comment]
In fact, I just need to update the `launch.sh` I used to generate the notebook `report_buildH.ipynb` and push the script to the repo.

**Comment by patrickfuchs** [comment]
Keep only the pdf file. Move all the files on Zenodo.

**Comment by patrickfuchs** [comment]
At the beginning I was thinking to update the validation procedure (notebook) with the new version of buildH. But it's a bad idea. It has been done in August 2019 with a version of buildH we can retrieve by jumping to the right commit. So it's better to report it as it was done at that time. Afterwards, we added tests and took care that the results generated with all new versions were stable. To keep things fully open and transparent, I deposited all the files and the report on Zenodo (https://zenodo.org/record/4715962).
So I renamed this issue "cleaning up", since we only have to clean up the directory and update the README.md.
